charts/app-config/values-integration.yaml

```diff
@@ -2517,6 +2517,8 @@ govukApplications:
         types:
           - command: ['bin/rake', 'document_sync_worker:run']
             name: document-sync-worker
+        redis:
+          enabled: true
```

Comment:
The indentation here needs to be one tab to the left, so it's not nested under `workers`.

Reply:
Sorted, thanks for spotting that 👍